### DISCUSSION TITLE
Valoriser le travail des cantines avec un badge 'amélioré'

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#1398] [Idées]
Il y a des cantines qui sont déçues parce que leurs données ne sont pas pour l'instant. Mais on veut encourager le transparence de données et aussi les améliorations alors ça pourrait un façon de les faire.

(imaginez que les chiffres ne sont pas bons - bien sur on garde le badge coloré quand le mesure et remplit)

Des suggestions:
![Screenshot 2022-05-13 at 11 23 59](https://user-images.githubusercontent.com/9282816/168253903-e8a5cef8-2dd9-405c-a0c4-f370052b54e3.png)
![Screenshot 2022-05-13 at 11 23 02](https://user-images.githubusercontent.com/9282816/168253909-b3894ca9-cc7e-4ba6-a733-3a55f7df68d0.png)
![Screenshot 2022-05-13 at 11 19 49](https://user-images.githubusercontent.com/9282816/168253913-15c8abb4-5abc-4cbe-9ee7-bb0806ea2ccc.png)
![Screenshot 2022-05-13 at 11 19 32](https://user-images.githubusercontent.com/9282816/168253915-df9119ad-f8b6-4268-97e6-08fc5ff8de07.png)

**Comment by valeriemerle** [comment]
Je trouve que c'est une bonne idée de montrer qu'il y a un progrès. Pour ça il faut au moins deux années pour comparer? Pour le symbole, la flèche (le plus simple) est pas mal? ou alors une animation, symbole qui clignote? trop compliqué peut etre...

**Reply by hfroot** [reply]
Oui il faut avoir les deux années
Je suis d'acc avec la flèche - pas convaincue par l'animation par contre - quand il y a plusieurs sur plusieurs cartes sur la même page ça pourrait être un peu trop même si c'est ludique :)
